Added Link for the Optimade tutorial on the 7th and 8th of september. (#44)
Materials-Consortia/materials-consortia.github.io: index.md

`dev[at]optimade.org` to register your interest.
The next meeting will take place on {% include {{ page.next_meeting }} %}.

> On the 7th and 8th of September 2021, there is a tutorial on how to use the Optimade API.

Reviewer: ```suggestion
On the 7th and 8th of September 2021, there will be an online tutorial on how to use the OPTIMADE API.
```
